src/display/Drawer/SideSheet.tsx

```diff
@@ -109,10 +109,17 @@ const StyledDialog = styled(Dialog)`
   flex-direction: column;
 `;
 
+const StyledHeading = styled(Heading)`
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+  font-size: 1rem;
+`;
+
 type SideSheetProps = {
   isOpen: boolean;
   openFrom: "left" | "right";
-  header?: React.ReactNode;
+  header: React.ReactNode;
```

Comment:
same question
Why did you delete the nullable to the header ? the header is still nullable, right ?
In the stories, you didn't put header on props.